lib/api/build.rs

```diff
@@ -85,14 +85,26 @@ fn main() {
             .define("WAMR_DISABLE_HW_BOUND_CHECK", "1")
             .define("WAMR_BUILD_TARGET", target_arch);
 
-        if cfg!(target_os = "windows") {
+        if target_os == "windows" {
             dst.define("CMAKE_CXX_COMPILER", "cl.exe");
             dst.define("CMAKE_C_COMPILER", "cl.exe");
             dst.define("CMAKE_LINKER_TYPE", "MSVC");
             dst.define("WAMR_BUILD_PLATFORM", "windows");
             dst.define("WAMR_BUILD_LIBC_UVWASI", "0");
         }
 
+        if target_os == "ios" {
+            // XXX: Hacky..
```

Comment:
Can you open an issue on the wamr repo and link it here.
Also, add a better comment on why we need to do this

Reply:
Sure, here it is: https://github.com/bytecodealliance/wasm-micro-runtime/pull/3889